packages/aws-rfdk/docs/upgrade/upgrading-0.38.md

```diff
@@ -0,0 +1,208 @@
+# Upgrading to RFDK v0.38.x or Newer
+
+Starting in RFDK v0.38.0, [Deadline Secrets Management](https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/secrets-management/deadline-secrets-management.html) will be enabled on the
+`Repository` and `RenderQueue` constructs by default when using Deadline 10.1.18 and later. This can cause some issues with existing farm configurations because Deadline Secrets Management requires:
```

Comment:
```suggestion
`Repository` and `RenderQueue` constructs by default when using Deadline 10.1.19 and later. This can cause some issues with existing farm configurations because Deadline Secrets Management requires:
```

Reply:
Updated